code/modules/mining/machine_vending.dm

```diff
@@ -30,6 +30,7 @@
 		new /datum/data/mining_equipment("Jump Boots",					/obj/item/clothing/shoes/bhop,										2000),
 		new /datum/data/mining_equipment("Proto-Kinetic Crusher",		/obj/item/twohanded/kinetic_crusher,								800),
 		new /datum/data/mining_equipment("Proto-Kinetic Accelerator",	/obj/item/gun/energy/kinetic_accelerator,							500),
+		new /datum/data/mining_equipment("Plasma Cutter",				/obj/item/gun/energy/plasmacutter,									1500),
```

Comment:
This short circuits mining tech/advanced mining tech?

Reply:
miners already have access to pka upgrades so why not a plasmacutter?